packages/Angular/explorer-core/src/lib/generic-browser-list/generic-browser-list.component.ts

```diff
@@ -83,10 +85,25 @@ export class GenericBrowserListComponent implements OnInit{
         .subscribe(() => this.filterItems(this.filter));
   }
 
-  public ngOnInit(): void {
+  public async ngOnInit(): Promise<void> {
     if(!this.disableAddButton){
       this.data.unshift({ text: this.resourceName });
     }
+
+    const md: Metadata = new Metadata();
+    const view = new RunView();
+    
+    const rvResult = await view.RunView({
+      EntityName: "Resource Types"
+    }, md.CurrentUser);
+
+    if(rvResult && rvResult.Success){
```

Comment:
@JS-BC this is unneeded - if you want entity object back just specify that to the RunView object, and it will be so :)